SharpBucket/V2/EndPoints/UsersEndpoint.cs

```diff
@@ -18,6 +18,12 @@ public class UsersEndpoint : EndPoint
             _repositoriesUrl = "repositories/" + accountName + "/";
         }
 
+        public UsersEndpoint(string accountName, ISharpBucketV2 sharpBucketV2) :
+            base(sharpBucketV2, "users/" + accountName + "/")
```

Comment:
```suggestion
            this(accountName, sharpBucketV2)
```
on the first constructor